.github/workflows/ios-fastlane.yml

```diff
@@ -0,0 +1,166 @@
+name: "Fastlane: Build and release ios app to TestFlight"
+
+on:
+  workflow_call:
+    inputs:
+      coordinator_port_http:
+        required: true
+        type: string
+      esplora_endpoint:
+        required: true
+        type: string
+      coordinator_p2p_endpoint:
+        required: true
+        type: string
+      network:
+        required: true
+        type: string
+        description: "The target network for the release, e.g. mainnet/regtest"
+      tag:
+        required: true
+        description: "The branch, tag or SHA to checkout."
+        type: string
+      oracle_endpoint:
+        required: true
+        type: string
+      oracle_pubkey:
+        required: true
+        description: "The public key of the oracle. Must match with the public key returned at http://${oracle_endpoint}/oracle/publickey"
+        type: string
+      app_scheme:
+        required: true
+        description: "For regtest enter 'test' or mainnet enter 'Runner'"
+        type: string
```

Comment:
❓This is a bit weird :O Why do we need to pass it in if we already know the values based on the network?

Reply:
Fair, we could just write a bash script for that.